deploy-manage/deploy/self-managed/_snippets/clean-up-multinode.md

```diff
@@ -0,0 +1,9 @@
+Perform the following steps on each node in the cluster:
+
+1. Open `elasticsearch.yml` in a text editor.
+2. Comment out or remove the `cluster.initial_master_nodes` setting.
+3. Update the `discovery.seed_hosts` value so it contains the IP address and port of each of the master-eligible {{es}} nodes in the cluster.
```

Comment:
> 3. Update the `discovery.seed_hosts` value so it contains the IP address and port of each of the master-eligible {{es}} nodes in the cluster.


Would it make sense to indicate that first node won't even have this setting? And the user has to add it.

The most important part of this section is the first node, although I like how it's explained as all nodes need to be reviewed.